package.json

```diff
@@ -1,63 +1,16 @@
 {
   "name": "aws-serverless-dev-portal-example",
-  "version": "1.0.0",
-  "description": "Example application for running a Serverless Developer Portal with API Gateway and Lambda",
+  "version": "2.0.0",
+  "description": "Serverless Developer Portal for API Gateway",
   "main": "lambda.js",
   "config": {
-    "artifactsS3Bucket": "YOUR_ARTIFACTS_BUCKET_NAME",
-    "siteS3Bucket": "YOUR_CLIENT_BUCKET_NAME",
-    "accountId": "YOUR_ACCOUNT_ID",
-    "primaryAwsRegion": "YOUR_PRIMARY_AWS_REGION",
-    "cloudFormationStackName": "YOUR_CLOUDFORMATION_STACK_NAME",
-    "apiGatewayApiName": "YOUR_API_GATEWAY_API_NAME",
-    "customersTableName": "DevPortalCustomers",
-    "identityPoolName": "DevPortal",
-    "apiGatewayApiId": "YOUR_API_GATEWAY_API_ID",
-    "cognitoRegion": "YOUR_COGNITO_REGION",
-    "cognitoUserPoolId": "YOUR_COGNITO_USER_POOL_ID",
-    "cognitoClientId": "YOUR_COGNITO_CLIENT_ID",
-    "cognitoIdentityPoolId": "YOUR_COGNITO_IDENTITY_POOL_ID",
-    "listenerLambdaFunctionName": "DevPortalSubscriptionListener",
     "marketplaceSubscriptionTopic": "arn:aws:sns:us-east-1:287250355862:aws-mp-subscription-notification-PRODUCT_CODE"
   },
   "scripts": {
-    "pre-config": "node scripts/pre-configure.js",
-    "post-config": "node scripts/post-configure.js",
-    "deconfig": "node -e \"require('./scripts/deconfigure.js')()\"",
-    "package": "npm run lambdas-npm-install && aws cloudformation package --template cloudformation/base.yaml --s3-bucket $npm_package_config_artifactsS3Bucket --output-template packaged-sam.yaml --region $npm_package_config_primaryAwsRegion",
```

Comment:
Which of these scripts can we delete? All of them?

Reply:
All of them. Some smalls scripts will need to be added, but we shouldn't need any of these.

Reply:
👍 If you don't get a chance to do that, I'll do it when I upgrade deps.